app/code/pagetype/Page.php

```diff
@@ -1,7 +1,41 @@
 <?php
 
 use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\TextareaField;
 
 class Page extends SiteTree
 {
+    /**
+     * Page database name
+     *
+     * @var string
+     */
+    private static $table_name = "PageField";
+
+    /**
+     * Page database
+     *
+     * @var array
+     */
+    private static $db = [
+        'Summary' => 'Text'
+    ];
+
+    /**
+     * Create fields in the page settings of the CMS
+     *
+     * @return void
+     */
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+        $fields->addFieldToTab(
+            'Root.Main',
+            TextareaField::create(
+                'Summary'
```

Comment:
Should there be a character limit on `Summary`? Can the user input a `summary` as large as a content page?

Reply:
Could be somethhing you confirm with the PO :)

Reply:
👍